src/bridge/settings/openedx/mfe/slot_config/mitx-staging/common-mfe-config.env.jsx

```diff
@@ -0,0 +1,175 @@
+import { PLUGIN_OPERATIONS, DIRECT_PLUGIN } from '@openedx/frontend-plugin-framework';
+import { getConfig } from '@edx/frontend-platform';
+import Footer, { Logo, MenuLinks, CopyrightNotice } from './Footer.jsx';
+
+const configData = getConfig();
+const currentYear = new Date().getFullYear();
+
+const userMenu = {
+    dashboard: {
+        url: `${configData.LMS_BASE_URL}/dashboard`,
+        title: 'Dashboard',
+    },
+    logout: {
+        url: `${configData.LMS_BASE_URL}/logout`,
+        title: 'Sign Out',
+    },
+}
+
+const DesktopHeaderUserMenu = (widget) => {
+  widget.content.menu = [
+    {
+      items: [
+        {
+          type: 'item',
+          href: userMenu.dashboard.url,
+          content: userMenu.dashboard.title,
+        },
+        {
+          type: 'item',
+          href:  userMenu.logout.url,
+          content: userMenu.logout.title,
+        },
+      ],
+    },
+  ];
+  return widget;
+};
+
+const LearningHeaderUserMenu = (widget) => {
+  widget.content.items = [
+    {
+      href: `${configData.LMS_BASE_URL}/dashboard`,
+      message: 'Dashboard',
+    },
+    {
+      href: `${configData.LMS_BASE_URL}/logout`,
+      message: 'Sign Out',
+    },
```

Comment:
We should use userMenu here as we used it in DesktopHeaderUserMenu. This goes for the config files.